### DISCUSSION TITLE
pp counter showing -2147483648 under certain conditions

### DISCUSSION THREAD

**Discussion by Ratchet0203** [ppy/osu#15483] [Q&A] [answered]
When Relax mod is enabled, you have a max combo of 1 and have gotten a 100 or a 50, the pp counter will display the 32-bit signed integer. I figured this out when I was about to play a map and had to adjust my mouse. I got a 100 on the first slider and I caught the bug out of the corner of my eye as I restarted. I've experimented further with the bug, and this may have to do with the Relax pp formula counting 100s and 50s as misses, as this doesn't occur without Relax, and having all misses doesn't break the counter either. I have attached an example video of me recreating this bug on another attempt.

https://user-images.githubusercontent.com/47193790/140449423-2825be89-2c5a-4d36-8f90-99d8ce4e0626.mp4

Only the conditions I've listed are consistent, however. The counter doesn't always break on the first object. Sometimes, it takes a moment, as shown below.

https://user-images.githubusercontent.com/47193790/140450786-72edc99a-dee0-41c7-ae3d-afc0c5888c3a.mp4

I've also experimented with other mods, and none of them, even classic (which changes the rules for how combo works), seem to change anything.

Lazer version: 2021.1105.0
Logs (as exiting map after recording second video):
[performance-input.log](https://github.com/ppy/osu/files/7480178/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/7480179/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/7480180/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7480181/updater.log)
[database.log](https://github.com/ppy/osu/files/7480182/database.log)
[network.log](https://github.com/ppy/osu/files/7480183/network.log)
[performance.log](https://github.com/ppy/osu/files/7480184/performance.log)
[performance-audio.log](https://github.com/ppy/osu/files/7480185/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/7480186/performance-draw.log)

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/15486
